frontend/app/src/main.tsx

```diff
@@ -27,6 +28,7 @@ if (import.meta.env.VITE_SENTRY_DSN) {
 
 ReactDOM.createRoot(document.getElementById('root')!).render(
   <QueryClientProvider client={queryClient}>
+    <ReactQueryDevtools initialIsOpen={false} />
```

Comment:
@mrkaye97 do we need to remove this for production builds?

Reply:
it's really nice for debugging btw if you haven't seen it - let's you restore loading states and such which makes e.g. lining up skeletons easy